scripts/ec2cleanup.py

```diff
@@ -218,8 +220,30 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_bucket(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name in bucket:
+                deleted_bucket = provider_mgmt.delete_s3_buckets(delete_all_objects=True,
+                                                                 bucket_names=[bucket])
+                if deleted_bucket:
+                    bucket_list.append([provider_name, bucket])
+            else:
+                logger.info("Smartstate Bucket is not found")
```

Comment:
REQUIRED: In addition, this else block looks weird. you will have a lot of such logging entries even for one iteration.
Are you sure you correctly indented this else block ?

Reply:
Missed Indentation, now updated code with right format.